src/components/Cards/ImageCard.js

```diff
@@ -1,31 +1,51 @@
 import React, { PropTypes } from 'react';
-import { Card } from '../UI';
+import { Card, CardMedia, CardTitle, CardText } from 'react-toolbox/lib/card';
 import styles from './ImageCard.css';
 
-export default class ImageCard extends React.Component {
-
-  render() {
-    const { onClick, onImageLoaded, image, text, description } = this.props;
-    return (
-      <Card onClick={onClick} className={styles.root}>
-        <img src={image} onLoad={onImageLoaded} />
-        <h2>{text}</h2>
-
-        {description ? <p>{description}</p> : null}
-      </Card>
-    );
-  }
-}
+const ImageCard = (
+  {
+    author,
```

Comment:
We can't assume entries have specific meta data.

Lots of entry types won't have an "author" field or "description" field.

Right now this is how the example content looks like:

<img width="280" alt="screen shot 2016-10-27 at 11 28 41 am" src="https://cloud.githubusercontent.com/assets/6515/19762047/98ebc86e-9c38-11e6-899c-c77592b1513a.png">

This obviously makes no sense. We need to figure out how to handle different content types well and show something that makes sense....